src/ui/server/helpers/sanitise-data/index.test.ts

```diff
@@ -53,6 +56,12 @@ describe('server/helpers/sanitise-data', () => {
         ADDRESS_LINE_2,
         POSTCODE,
         BROKER_NAME,
+        NAME,
+        REGISTRATION_NUMBER,
+        ADDRESS,
+        FIRST_NAME,
+        LAST_NAME,
+        POSITION,
```

Comment:
Should `POSITION` allow pure numbers?

Reply:
technically could be an edge case which would break the db call